doc/api/toolkits/axes_grid.rst

```diff
@@ -0,0 +1,21 @@
+.. _axes_grid-api-index:
+
+####################################
+The Matplotlib axes_grid Toolkit API
+####################################
+
+:Release: |version|
+:Date: |today|
+
+.. currentmodule:: mpl_toolkits
+
+Axes Grid
+---------
+
+.. autosummary::
+   :toctree: ../_as_gen
+   :template: automodule.rst
+
+   axes_grid
```

Comment:
I can remove this in the `api` page in this PR and include a link to the old documentation in previous 2.x iterations in the `axes_grid1` page. WDYT?